usr/share/rear/layout/save/default/445_guess_bootloader.sh

```diff
@@ -49,6 +49,15 @@ for block_device in /sys/block/* ; do
     bootloader_area_strings_file="$TMP_DIR/bootloader_area_strings"
     block_size=$( get_block_size ${disk_device##*/} )
     dd if=$disk_device bs=$block_size count=4 | strings >$bootloader_area_strings_file
+    # First check for Hah!IdontNeedEFI for GPT BIOS boot (for computers where UEFI could also be possible)
+    # Usually these use GRUB instead of EFI (issue #1752)
+    if grep -q "Hah!IdontNeedEFI" $bootloader_area_strings_file ; then
+        if grep -q -i GRUB $bootloader_area_strings_file ; then
+           LogPrint "Using guessed bootloader 'GRUB'"
+           echo "GRUB" >$bootloader_file
+           return
+        fi 
+    fi
```

Comment:
The below comments in lines 61,62,63 actually belong
before all the checks i.e. like this:
<pre>
dd if=$disk_device bs=$block_size count=4 | strings >$bootloader_area_strings_file
# Examine the strings in the first bytes on the disk to guess the used bootloader,
# see layout/save/default/450_check_bootloader_files.sh for the known bootloaders.
# Test the more specific strings first because the first match wins:
# First check for Hah!IdontNeedEFI for GPT BIOS boot (for computers where UEFI could also be possible)
</pre>